platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/crypto/CryptoStatic.java

```diff
@@ -626,6 +630,21 @@ public static Map<NodeId, KeysAndCerts> initNodeSecurity(
         return store;
     }
 
+    /**
+     * Generate a {@link BlsKeyPair} using a {@link SignatureSchema} and a {@link SecureRandom} instance
+     * @return a new {@link BlsKeyPair}
+     * @throws NoSuchAlgorithmException the algorithm is not supported
+     */
+    public static BlsKeyPair generateBlsKeyPair(@Nullable final SecureRandom secureRandom)
+            throws NoSuchAlgorithmException {
+        final SignatureSchema SIGNATURE_SCHEMA =
+                SignatureSchema.create(Curve.ALT_BN128, GroupAssignment.SHORT_SIGNATURES);
+        if (secureRandom == null) {
+            return BlsKeyPair.generate(SIGNATURE_SCHEMA);
```

Comment:
Nit: might be beneficial to add a debug log that the default secure random generator is being used here ?

Reply:
Added debug log statements.